pontoon/api/urls.py

```diff
@@ -14,6 +14,61 @@
         r"^graphql/?$",
         csrf_exempt(GraphQLView.as_view(schema=schema, graphiql=True)),
     ),
+    path(
+        "api/new/",
```

Comment:
This ought to be versioned more appropriately, i.e. something like
```suggestion
        "api/v2/",
```

Reply:
I was wondering what to name this thing :)